src/Resources/Resources/ChangeLog.md

```diff
@@ -30,6 +30,10 @@
     - Update-AzTag -ResourceId
 * Brought ScopedDeployment from SDK 3.3.0 
 * Breaking changes for policy cmdlets to support typed pipeline objects
+* Add Get-AzDeploymentWhatIf cmdlet for getting ARM template What-If at subscription scope.
```

Comment:
Please move new contents after "Upcoming Release", our tools for release depend on this structure

Reply:
Missed this one after resolving merge conflicts. Pushed out a fix.